helix-term/src/keymap/default.rs

```diff
@@ -205,6 +205,7 @@ pub fn default() -> HashMap<Mode, Keymap> {
             "f" => file_picker,
             "F" => file_picker_in_current_directory,
             "b" => buffer_picker,
+            "m" => mru_buffer_picker,
```

Comment:
I agree with @the-mikedavis here, sorting the buffer picker by MRU sounds like good default behavior.